sdk/identity/azure-identity/samples/README.md

```diff
@@ -4,7 +4,7 @@ languages:
   - python
 products:
   - azure
-  - azure-identity
```

Comment:
Is an `identity` tag necessary for these to show up with Identity samples?

Reply:
I believe so but question whether we want these samples to show up there. Today there's no "there" there; the [sample browser](https://docs.microsoft.com/samples/browse/) has nothing for "identity". The sample browser covers products outside Azure, so not knowing the intended scope of "identity", I'm reluctant to make our samples the only ones in that category.